### PR TITLE
maint: update Refinery app version to 2.4.3

### PR DESCRIPTION
**Pull request by robbkidd**
## Which problem is this PR solving?

- Updating Refinery chart to deploy [Refinery v2.4.3](https://github.com/honeycombio/refinery/releases/tag/v2.4.3)

### CODE REVIEW

**Review by kentquirk — APPROVED** [review]
👍

**Review by MikeGoldsmith — APPROVED** [review]

**Comment by robbkidd** [comment]
On hold, pending a Refinery 2.4.3 release containing a fix for the fix in 2.4.2 (ETA: today).

**Comment by TylerHelmuth** [comment]
I agree with skipping a 2.4.2 bump and jumping straight to 2.4.3

File: charts/refinery/Chart.yaml [hunk 1]
```diff
@@ -2,8 +2,8 @@ apiVersion: v2
 name: refinery
 description: Chart to deploy Honeycomb Refinery
 type: application
-version: 2.8.1
-appVersion: 2.4.1
+version: 2.8.2
+appVersion: 2.4.2
```
Comment (robbkidd) [inline charts/refinery/Chart.yaml:6]: Skip 2.4.2 because there's a regression in there; 2.4.3 has the fix.

```suggestion
appVersion: 2.4.3
```

**Review by TylerHelmuth — APPROVED** [review]

**Pull request merged**